src/Tickets/Commerce/Cart.php

```diff
@@ -15,6 +20,8 @@
  */
 class Cart {
 
+	use Cart_Trait;
+
 	/**
 	 * Which URL param we use to identify a given page as the cart.
```

Comment:
This file needs the `@since` to be updated in the docblocks. I see a lot of them not touched.

Reply:
Thanks, I added updated `@since` tags to things that had notable changes.